src/brpc/builtin/common.cpp

```diff
@@ -71,6 +71,7 @@ inline bool url_wildcmp(const char* wild, const char* str) {
             ++str;
         } else {
             wild = mp;
+            // Is it guaranteed that cp is not nullptr always here?
```

Comment:
yes. If 'wild' does not contain *, *str should be 0 after the first loop and this while loop should be jumped over; If 'wild'  has a *, cp should be set (to str+1) before here.